From 92606b5e0000c25f5daae6c17b0ab71e9fb4c3b4 Mon Sep 17 00:00:00 2001
From: Stefan Wahren <wahrenst@gmx.net>
Date: Wed, 25 Dec 2019 15:55:29 +0100
Subject: [PATCH] ARM: dts: bcm2838: Backport node names from upstream

According to devicetree specification the node name should describe
the general class of device like ethernet or interrupt-controller.

Signed-off-by: Stefan Wahren <wahrenst@gmx.net>
---
 arch/arm/boot/dts/bcm2838.dtsi | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

--- a/arch/arm/boot/dts/bcm2838.dtsi
+++ b/arch/arm/boot/dts/bcm2838.dtsi
@@ -27,7 +27,7 @@
 			reg = <0x40000000 0x100>;
 		};
 
-		gicv2: gic400@40041000 {
+		gicv2: interrupt-controller@40041000 {
 			interrupt-controller;
 			#interrupt-cells = <3>;
 			compatible = "arm,gic-400";
@@ -346,7 +346,7 @@
 			status = "okay";
 		};
 
-		genet: genet@7d580000 {
+		genet: ethernet@7d580000 {
 			compatible = "brcm,genet-v5";
 			reg = <0x0 0x7d580000 0x10000>;
 			status = "okay";
@@ -362,7 +362,7 @@
 				compatible = "brcm,genet-mdio-v5";
 				reg = <0xe14 0x8>;
 				reg-names = "mdio";
-				phy1: genet-phy@0 {
+				phy1: ethernet-phy@0 {
 					compatible =
 						"ethernet-phy-ieee802.3-c22";
 					/* No interrupts - use PHY_POLL */
